go/libkb/env.go

```diff
@@ -187,6 +187,9 @@ type TestParameters struct {
 	// set to true to use production run mode in tests
 	UseProductionRunMode bool
 
+	// Set to override the Keybase API server URI in a test.
+	ServerURI string
```

Comment:
do you need this or is it sufficient to just set the URI via env?

Reply:
Made redundant by the First...WithSkip wrapper that I was not aware of, I will need kbweb access to make this work locally then.

Reply:
Removed.